molgenis/molgenis
# M4765 As MOLGENIS user I want to have a persistent URL for each instance of an entity

#### Checklist

## Test
- In the dataexplorer settings set standalone reports to true
- Go to Entity table and view the entityreport for any row
- Copy the URL
- Paste the URL in your browser and see default report
- In the Freemarker template, create a view-standalone-report-specific-sys_md_EntityType.ftl with some nice HTML
- Go to Entity table and view the entityreport for any row
- Copy the URL
- Paste the URL in your browser and see nice HTML
